staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -61,6 +62,9 @@ const (
 	defaultBatchBufferSize = 1000        // Buffer up to 1000 events before blocking.
 	defaultBatchMaxSize    = 100         // Only send 100 events at a time.
 	defaultBatchMaxWait    = time.Minute // Send events at least once a minute.
+
+	defaultBatchMaxRoutine      = 10 //Send routine at 10 qos.
```

Comment:
`10 qps`?

Also, need an extra space between `//` and a comment

Reply:
yes, typo